cw-orch-daemon/src/core.rs

```diff
@@ -144,6 +146,44 @@ impl DaemonAsync {
         Ok(result)
     }
 
+    /// Instantiate a contract.
+    pub async fn instantiate2<I: Serialize + Debug>(
+        &self,
+        code_id: u64,
+        init_msg: &I,
+        label: Option<&str>,
+        admin: Option<&Addr>,
+        coins: &[Coin],
+        salt: Binary,
+    ) -> Result<CosmTxResponse, DaemonError> {
+        let sender = &self.sender;
+
+        let init_msg = MsgInstantiateContract2 {
+            code_id,
+            label: label.unwrap_or("instantiate_contract").to_string(),
+            admin: admin.map(|a| a.to_string()).unwrap_or("".to_string()),
+            sender: sender.address()?.to_string(),
+            msg: serde_json::to_vec(&init_msg)?,
+            funds: proto_parse_cw_coins(coins)?,
+            salt: salt.to_vec(),
+            fix_msg: false,
+        };
+
+        let result = sender
+            .commit_tx_any(
+                vec![Any {
+                    type_url: "/cosmwasm.wasm.v1.MsgInstantiateContract2".to_string(),
```

Comment:
Are there any advantages for keeping it raw instead  MsgInstantiateContract2::TYPE_URL/MsgInstantiateContract2::type_url() (depending on a version)

Reply:
There is no Type url definition on this one unfortunately